ecommerce/views.py

```diff
@@ -341,12 +342,19 @@ def get(self, request, *args, **kwargs):
         promo_coupons = (
             Coupon.objects.annotate(latest_coupon_type=latest_coupon_type_subquery)
             .filter(latest_coupon_type=CouponPaymentVersion.PROMO)
+            .select_related("payment")
```

Comment:
As discussed over the call, we can prefetch payment versions but that will require us to update the latest_version property.